pkg/api/entries.go

```diff
@@ -438,3 +427,39 @@ func SearchLogQueryHandler(params entries.SearchLogQueryParams) middleware.Respo
 
 	return entries.NewSearchLogQueryOK().WithPayload(resultPayload)
 }
+
+var ErrNotFound = errors.New("grpc returned 0 leaves with success code")
+var ErrUnexpected = errors.New("unexpected error")
```

Comment:
I don't think we need this `ErrUnexpected` since we don't check for it anywhere

Reply:
Okay cool, I took it out!